### PR TITLE
Readonly internal subscription form for contracted enterprise clients

### PR DESCRIPTION
**Pull request by nickpell**
http://manage.dimagi.com/default.asp?188456

screenshot:
screen shot 2015-12-02 at 5 02 56 pm

### CODE REVIEW

File: corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/field_with_text.html [hunk 1]
```diff
@@ -9,7 +9,6 @@
         </label>
         <div class="controls" style="padding-top: 5px;">
             <input type="hidden"
-                   name="{{ field.html_name }}"
```
Comment (nickpell) [inline corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/field_with_text.html]: this is to prevent from displaying like this:
screen shot 2015-12-02 at 5 04 31 pm

notifying @biyeun in case this is a problem, but it didn't seem to be with any of the other field types on this page

**Comment by nickpell** [comment]
@kaapstorm

**Comment by kaapstorm** [comment]
I'm happy with this, and preventing "[n@n.com]n@n.com" looks like the right decision to me, but I'll hold off on merge for @biyeun to confirm.

File: corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/field_with_text.html (hunk 1 above)
Comment (nickpell) [inline corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/field_with_text.html]: @biyeun thoughts?

Comment (nickpell) [inline corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/field_with_text.html]: @orangejenny Is this related to https://github.com/dimagi/commcare-hq/pull/9395 ?

Comment (esoergel) [inline corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/field_with_text.html]: I'm also concerned that this will result in the value not being submitted to the server (since the 'name' attr is what's used to identify the value)

Comment (orangejenny) [inline corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/field_with_text.html]: > @orangejenny Is this related to #9395 ?

I don't think so...but I do agree with Ethan that this seems like it's going to prevent the value from getting saved.

**Comment by nickpell** [comment]
@biyeun bump

File: corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/field_with_text.html (hunk 1 above)
Comment (biyeun) [inline corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/field_with_text.html]: @nickpell Did you try not making it a select 2 widget?

**Comment by biyeun** [comment]
@nickpell maybe https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/accounting/static/accounting/js/accounting.billing_info_handler.js#L13 ?

**Comment by nickpell** [comment]
that looks like the place

**Comment by biyeun** [comment]
@nickpell just make sure that's not actually being used anywhere else tho...heh

**Comment by nickpell** [comment]
fixed!

**Comment by biyeun** [comment]
:+1: waiting for build then I think this is g2g

**Comment by nickpell** [comment]
:white_check_mark:
